gef.py

```diff
@@ -4323,7 +4323,9 @@ def do_invoke(self, argv):
             extra = "dirty" if len(subprocess.check_output('git ls-files -m', cwd=gef_dir, shell=True).decode("utf8").strip()) else "clean"
             gef_print("GEF: rev:{} (Git - {})".format(ver, extra))
         else:
+            gef_blob_hash = subprocess.check_output("git hash-object {}".format(gef_fpath), shell=True).decode().strip()
             gef_print("GEF: (Standalone)")
+            gef_print("Blob Hash({}): {}".format(gef_fpath, gef_blob_hash))
```

Comment:
Maybe add to our issue template how to use this?

Reply:
Usage hasn't changed in my opinion. The Blob Hash is only printed in `standalone` mode and should be included when somebody opens an issue.

Or do you mean a little documentation on how the commit can be found using this blob hash from a developers point of view?

Reply:
Yes, I mean even a comment for the issue writer to maybe find the commit using this.

It's not really on them, but I am afraid that I myself will forget how to use this information. Maybe a blurb somewhere on a debugging doc, even.

Or maybe I should just remember how. 👀

Reply:
ok, i'll just add the `version` command to the documentation then